spring-cloud-gcp-data-firestore/src/test/java/com/google/cloud/spring/data/firestore/it/FirestoreIntegrationTests.java

```diff
@@ -221,8 +222,8 @@ public void saveAllTest() {
 	@Test
 	public void saveAllBulkTest() {
 		Flux<User> users = Flux.create(sink -> {
-			for (int i = 0; i < 1000; i++) {
-				sink.next(new User("testUser " + i, i));
+			for (int i = 0; i < 10000; i++) {
```

Comment:
We can probably keep it at 1,000. 10K was useful for weeding out flakes, but likely not necessary for a regularly run test.